packages/analytics-core/src/config.ts

```diff
@@ -52,7 +52,7 @@ export class Config implements IConfig {
   transportProvider: Transport;
   storageProvider?: Storage<Event[]>;
   useBatch: boolean;
-  request_metadata?: RequestMetadata;
+  requestMetadata?: RequestMetadata;
```

Comment:
Why are we exposing this in the config?

Reply:
Because `client.config` is the only one thing we are passing to core destination. I added it hidden options so that it won't expose to customers at https://github.com/amplitude/Amplitude-TypeScript/pull/772/commits/0bb2a8f5e124784ff79afb9394f2dfc0b57096e7